.changeset/heavy-hornets-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/design-system': patch
+---
+
+Fix `useHook` to keep the returned current selected theme name in sync with the provider.
```

Comment:
What's `useHook`? 🤔

Reply:
Fixed in this commit: d0f30d5cfefdd6fdb2ceb0de59b65b47e0a78f62